frontend/src/components/admin/CoursesOverviewTab.tsx

```diff
@@ -12,35 +17,49 @@ const CoursesOverviewTab = (): React.ReactElement => {
       modules: [
         {
           title: "Module 1",
-          description: "Test1",
           published: true,
+          imageLink: DEFAULT_IMAGE,
         },
         {
           title: "Module 2",
-          description: "Test2",
           published: false,
+          imageLink: DEFAULT_IMAGE,
         },
         {
           title: "Module 3",
-          description: "Test3",
           published: true,
+          imageLink: DEFAULT_IMAGE,
         },
       ],
     },
   ];
 
   return (
-    <div>
-      {dummyCourses.map((x, i) => (
-        <CoursePreview
-          key={i}
-          title={x.title}
-          description={x.description}
-          isPrivate={x.isPrivate}
-          modules={x.modules}
-        />
-      ))}
-    </div>
+    <Box my={6} mx={9} flex="1">
+      <Flex
+        justify="space-between"
+        borderBottom="1px"
+        borderColor="background.lightgrey"
+      >
+        <Text variant="display-lg" pb={6}>
+          Courses
+        </Text>
+        <Button variant="md" leftIcon={<SmallAddIcon />}>
+          Create New Course
+        </Button>
+      </Flex>
+      <VStack spacing={12} mt={6}>
+        {dummyCourses.map((x, i) => (
```

Comment:
nitpick: I feel like dummy data should always be at the API Client level, just so that any changes that need to be made when we switch over are centralized at one level. This would mean creating a function in the API client called `getDummyCourses(...)` that behaves the same as a real course request (i.e async/await behaviour, same response format, same errors). It would ideally make integration easier.

Reply:
This is a good point. I don't think it's worth changing for this PR (and mine frankly), but for future tickets we should do this.

Reply:
Can always do a quick fix PR for moving all the dummy data together